.cirrus.yml

```diff
@@ -37,14 +37,22 @@ task:
       test_script:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh test
-    - name: analyze
+    - name: analyze_master
       env:
         matrix:
           CHANNEL: "master"
-          CHANNEL: "stable"
       script:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh analyze
+    ## TODO(cyanglaz):
+    ## Combing stable and master analyze jobs when integration test null safety is ready on flutter stable.
+    - name: analyze_stable
+      env:
+        matrix:
+          CHANNEL: "stable"
+      script:
+        - flutter channel $CHANNEL
+        - ./script/incremental_build.sh analyze --exclude "google_sign_in_web, connectivity_for_web, file_selector_web, google_maps_flutter_web, url_launcher_web"
```

Comment:
```suggestion
      script:
        - flutter channel $CHANNEL
        - find . -type d -wholename '*_web/example' -exec rm -rf {} \; # Remove web example dirs
        - ./script/incremental_build.sh analyze
```

?

Reply:
if we remove, the example folder will not be available for all the tests in the same task right? It might be dangerous since it's hard to notice.

Reply:
Ah I didn't know. However, it's probably for the best, since any task that attempts to pub get inside the example in stable will probably hang, right? This should be deleted as early as possible, I guess? Maybe in more places? :P

Reply:
BTW, do the tasks run always in the same order?

Reply:
oh never mind, they are actually independent. So yeah removing is fine, let me do that